lib/dependencies.js

```diff
@@ -46,7 +46,10 @@ let versionMap = {
   "minimatch": "^3.0.2",
   "through2": "^2.0.1",
   "tslint": "^3.11.0",
-  "typings": "^1.3.0",
+  "@types/protractor": "^1.5.20",
+  "@types/jasmine": "^2.5.35",
+  "@types/selenium-webdriver": "^2.53.33",
```

Comment:
Could you test this?

In my projects I have to lock this to version `"@types/selenium-webdriver": "2.44.26",`